Sources/MapLibreSwiftUI/MapView.swift

```diff
@@ -103,13 +103,16 @@ public struct MapView<T: MapViewHostViewController>: UIViewControllerRepresentab
 
         if cameraDisabled == false {
             context.coordinator.updateCamera(mapView: uiViewController.mapView,
-                                             camera: $camera.wrappedValue,
+                                             camera: camera,
                                              animated: isStyleLoaded)
         }
     }
 
     @MainActor private func applyModifiers(_ mapViewController: T, runUnsafe: Bool) {
-        mapViewController.mapView.contentInset = mapViewContentInset
+        if let mapViewContentInset {
+            mapViewController.mapView.automaticallyAdjustsContentInset = false
```

Comment:
So, here's where things get a bit funny. We HAVE to disable automatic inset adjustment here, but as initially submitted, this PR does NOT have a solution for opting back in once you've disabled it. It feels like the API for doing so would be a bit clunky.

For example, we could add a modifier which sets this explicitly, but then it's going to sometimes conflict with this and so it won't take effect.

A second approach might be to model content insets as an enum. I _think_ that works.... Is it really a binary choice of full manual or full auto?